topics/admin/tutorials/maintenance/slides.html

```diff
@@ -2,98 +2,163 @@
 layout: tutorial_slides
 logo: "GTN"
 
-title: "Server Maintenance"
+title: "Server Maintenance and Backups"
 zenodo_link: ""
 questions:
-  - How to maintain a Galaxy service?
+  - How do I maintain a Galaxy server?
+  - What happens if I lose everything?
 objectives:
   - Learn about different maintenance steps
-time_estimation: "1h"
+  - Learn what to back up and how to recover
+time_estimation: "30m"
 key_points:
-  - Remember to back Galaxy up
-  - Use Ansible
+  - Use configuration management (e.g. Ansible)
+  - Back up the parts of Galaxy that can't be recreated
 contributors:
   - natefoo
   - bgruening
   - slugger70
   - hexylena
 ---
+class: left
 
 # Runaway Storage
 
 Tips:
 - Set quotas
-- `tmpwatch` your job working directory
+- `tmpwatch` (`tmpreaper` on Debian-based distros) your job working directory
   - `cleanup_job` in `galaxy.yml` (defaults to `always` though)
```

Comment:
I normally change it to `onsuccess` to be able to debug user issue reports.

Reply:
Same, which is why you might need `tmpwatch` (you don't if you leave `cleanup_job` at the default).